### PR TITLE
r_cons: Save and restore the active code page on Windows

### PR DESCRIPTION
**Pull request by kazarmy**
**Your checklist for this pull request**
- [X] I've read the [guidelines for contributing](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md) to this repository
- [X] I made sure to follow the project's [coding style](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md#code-style)
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [radare2 book](https://github.com/radareorg/radare2book) with the relevant information (if needed)

**Detailed description**

This pr fixes the following:

chcp_change

**Test plan**

On the Windows command line:
```
> chcp
> r2 -
> chcp
```

Before and after active code page should be the same.

**Closing issues**

...

### CODE REVIEW

File: binr/r2r/r2r.c [hunk 1]
```diff
@@ -466,12 +466,14 @@ int main(int argc, char **argv) {
...
-	(void)SetConsoleOutputCP (old_cp);
-	// chcp doesn't pick up the code page switch for some reason
-	char *chcp = r_str_newf ("chcp %u > NUL", old_cp);
-	if (chcp) {
-		system (chcp);
-		free (chcp);
+	if (old_cp) {
+		(void)SetConsoleOutputCP (old_cp);
+		// chcp doesn't pick up the code page switch for some reason
+		char *chcp = r_str_newf ("chcp %u > NUL", old_cp);
```
Comment (ret2libc) [inline binr/r2r/r2r.c:472]: `r_sys_cmdf` instead of this?

File: libr/cons/cons.c [hunk 2]
```diff
@@ -578,6 +579,15 @@ R_API RCons *r_cons_new() {
 R_API RCons *r_cons_free() {
 #if __WINDOWS__
 	r_cons_enable_mouse (false);
+	if (I.old_cp) {
+		(void)SetConsoleOutputCP (I.old_cp);
+		// chcp doesn't pick up the code page switch for some reason
+		char *chcp = r_str_newf ("chcp %u > NUL", I.old_cp);
```
Comment (ret2libc) [inline libr/cons/cons.c:585]: `r_sys_cmdf`

File: binr/r2r/r2r.c (hunk 1 above)
Comment (kazarmy) [reply binr/r2r/r2r.c]: ok 1f27394f002d8faa32846ff36cc7c85c70ecb5d5

File: libr/cons/cons.c (hunk 2 above)
Comment (kazarmy) [reply libr/cons/cons.c]: ok 1f27394f002d8faa32846ff36cc7c85c70ecb5d5

**Pull request merged**